spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -989,10 +989,7 @@ function timeoutPacket(
       // NOTE: For ORDERED_ALLOW_TIMEOUT, the relayer must first attempt the receive on the destination chain
       // before the timeout receipt can be written and subsequently proven on the sender chain in timeoutPacket
       case ORDERED_ALLOW_TIMEOUT:
-        // ordered channel: check that packet has not been received
-        // only allow timeout on next sequence so all sequences before the timed out packet are processed (received/timed out)
-        // before this packet times out
-        abortTransactionUnless(nextSequenceRecv == packet.sequence)
+        abortTransactionUnless(nextSequenceRecv > packet.sequence)
```

Comment:
For both packet timeouts and for timeout on close the `nextSequenceRecv` on the counterparty would have been incremented by one before the timeout receipt is written.

Reply:
I believe this was correct as it was

Reply:
The reason for changing it is that in line 709 inside `recvPacket` the `nextSequenceRecv` is incremented by 1 before the timeout receipt is written:

```
nextSequenceRecv = nextSequenceRecv + 1
provableStore.set(nextSequenceRecvPath(packet.destPort, packet.destChannel), nextSequenceRecv)
```

So if I relayer tries to timeout the packet, the `nextSequenceRecv` in the timeout message will be > than `packet.sequence`, won't it?

Same thing for the condition checked in `timeoutOnClose`.

Is this reasoning correct, @AdityaSripal?

Reply:
@AdityaSripal for timing out a packet if the channel is open, should the condition still be that the relayer must attempt to receive the packet (as the comment above this line says)? If that's the case, then I guess it's ok to not support the situation where `packet.sequence` is >= `nextSequenceRecv`.